docresources/SyncBot/syncbot-example.sh

```diff
@@ -7,16 +7,9 @@ rm -rf ./build
 gulp build
 
 export PROCBOT_BOTS_TO_LOAD="syncbot"
-
+export CONFIG_TO_LOAD="syncbot_test"
```

Comment:
This should be `SYNCBOT_CONFIG_TO_LOAD`, else it doesn't fit into the current configuration file paradigm.